src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.jiraactions.api.page
```

Comment:
If it's not API we can keep it outside of the `api` package.

Reply:
This class will be part of the API.